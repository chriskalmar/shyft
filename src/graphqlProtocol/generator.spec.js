// import { generateGraphQLSchema } from './generator';

describe.skip('generator', () => {
  describe('test', () => {
    it('should ...', () => {});
  });
});
