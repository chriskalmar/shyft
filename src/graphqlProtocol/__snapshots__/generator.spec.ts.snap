// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generator test should render GraphQL Schema 1`] = `
GraphQLSchema {
  "__validationErrors": undefined,
  "_directives": Array [
    "@include",
    "@skip",
    "@deprecated",
    "@specifiedBy",
  ],
  "_implementationsMap": Object {
    "Node": Object {
      "interfaces": Array [],
      "objects": Array [
        "TestEntityName",
      ],
    },
  },
  "_mutationType": "Mutation",
  "_queryType": "Query",
  "_subTypeMap": Object {},
  "_subscriptionType": "Subscription",
  "_typeMap": Object {
    "Boolean": "Boolean",
    "CreateTestEntityNameInput": "CreateTestEntityNameInput",
    "CreateTestEntityNameInstanceInput": "CreateTestEntityNameInstanceInput",
    "CreateTestEntityNameInstanceNestedInput": "CreateTestEntityNameInstanceNestedInput",
    "CreateTestEntityNameNestedInput": "CreateTestEntityNameNestedInput",
    "CreateTestEntityNameOutput": "CreateTestEntityNameOutput",
    "Cursor": "Cursor",
    "DeleteTestEntityNameByIdInput": "DeleteTestEntityNameByIdInput",
    "DeleteTestEntityNameInput": "DeleteTestEntityNameInput",
    "DeleteTestEntityNameOutput": "DeleteTestEntityNameOutput",
    "ID": "ID",
    "Int": "Int",
    "Mutation": "Mutation",
    "Node": "Node",
    "OnCreateTestEntityNameInput": "OnCreateTestEntityNameInput",
    "OnCreateTestEntityNameInstanceInput": "OnCreateTestEntityNameInstanceInput",
    "OnCreateTestEntityNameInstanceNestedInput": "OnCreateTestEntityNameInstanceNestedInput",
    "OnCreateTestEntityNameNestedInput": "OnCreateTestEntityNameNestedInput",
    "OnCreateTestEntityNameOutput": "OnCreateTestEntityNameOutput",
    "OnDeleteTestEntityNameInput": "OnDeleteTestEntityNameInput",
    "OnDeleteTestEntityNameOutput": "OnDeleteTestEntityNameOutput",
    "OnUpdateTestEntityNameInput": "OnUpdateTestEntityNameInput",
    "OnUpdateTestEntityNameInstanceInput": "OnUpdateTestEntityNameInstanceInput",
    "OnUpdateTestEntityNameInstanceNestedInput": "OnUpdateTestEntityNameInstanceNestedInput",
    "OnUpdateTestEntityNameNestedInput": "OnUpdateTestEntityNameNestedInput",
    "OnUpdateTestEntityNameOutput": "OnUpdateTestEntityNameOutput",
    "PageInfo": "PageInfo",
    "Query": "Query",
    "String": "String",
    "Subscription": "Subscription",
    "TestEntityName": "TestEntityName",
    "TestEntityNameConnection": "TestEntityNameConnection",
    "TestEntityNameEdge": "TestEntityNameEdge",
    "TestEntityNameFilter": "TestEntityNameFilter",
    "TestEntityNameOrderBy": "TestEntityNameOrderBy",
    "UpdateTestEntityNameByIdInput": "UpdateTestEntityNameByIdInput",
    "UpdateTestEntityNameInput": "UpdateTestEntityNameInput",
    "UpdateTestEntityNameInstanceInput": "UpdateTestEntityNameInstanceInput",
    "UpdateTestEntityNameInstanceNestedInput": "UpdateTestEntityNameInstanceNestedInput",
    "UpdateTestEntityNameNestedInput": "UpdateTestEntityNameNestedInput",
    "UpdateTestEntityNameOutput": "UpdateTestEntityNameOutput",
    "__Directive": "__Directive",
    "__DirectiveLocation": "__DirectiveLocation",
    "__EnumValue": "__EnumValue",
    "__Field": "__Field",
    "__InputValue": "__InputValue",
    "__Schema": "__Schema",
    "__Type": "__Type",
    "__TypeKind": "__TypeKind",
  },
  "astNode": undefined,
  "description": undefined,
  "extensionASTNodes": undefined,
  "extensions": undefined,
}
`;
